src/qt/intro.cpp

```diff
@@ -349,7 +349,7 @@ void Intro::setSizeWarningLabel()
     requiredSpace = m_blockchain_size;
     QString storageRequiresMsg = QObject::tr("At least %1 GB of data will be stored in this directory, and it will grow over time.");
     if (ui->prune->isChecked()) {
-        uint64_t prunedGBs = std::ceil(pruneTarget * 1024 * 1024.0 / GB_BYTES);
+        uint64_t prunedGBs = std::ceil(m_prune_target * 1000 * 1000 / GB_BYTES);
```

Comment:
This is inconsistent with https://github.com/bitcoin/bitcoin/blob/ec3ed5a4487886f1c2a35fda0a3289be7b280248/src/init.cpp#L386-L388

Reply:
Ok, then I will change everything to Mebibyte

Reply:
MiB is indeed consistent with other places, but you should also change `uint64_t GB_BYTES{1000000000}` to `uint64_t GiB_BYTES{1073741824}`.

Reply:
> MiB is indeed consistent with other places, but you should also change `uint64_t GB_BYTES{1000000000}` to `uint64_t GiB_BYTES{1073741824}`. But that impacts the settings screen. Probably better to leave this change for another PR.

I'm not sure what the goal of this change would be, and would definitely think it should be done as separately from this PR. The existing conversions in this PR as of f3108a01a03a937fc3d31f007caa69e1f95f182f look correct to me, and I'd expect normal users to be more familiar with GB than GiB. Also if a user sees a GiB value and mistakenly interprets it as a GB value their node will use more storage than expected and maybe lead to problems.